cmd/read_only_geth/config.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"github.com/ethereum/go-ethereum/cmd/utils"
+	"github.com/ethereum/go-ethereum/eth/ethconfig"
+	"github.com/ethereum/go-ethereum/internal/ethapi"
+	"github.com/ethereum/go-ethereum/node"
+)
+
+func makeFullNode(datadir string) (*node.ReadOnlyNode, ethapi.Backend) {
+	stack, cfg := makeConfigNode(datadir)
+	backend, _ := utils.RegisterReadOnlyEthService(stack, cfg)
+	return stack, backend
+}
+
+func makeConfigNode(datadir string) (*node.ReadOnlyNode, *ethconfig.Config) {
+	node_cfg := defaultNodeConfig()
+	eth_cfg := ethconfig.Defaults
```

Comment:
The rest of the codebase uses `camelCase` instead of `snake_case`. Please be consistent and switch everything to camel.